src/demo/pages/ListPage/examples/List.Grid.Example.tsx

```diff
@@ -35,44 +34,32 @@ export class ListGridExample extends React.Component<IListGridExampleProps, any>
           items={ this.props.items }
           getItemCountForPage={ this._getItemCountForPage }
           getPageHeight={ this._getPageHeight }
+          renderedWindowsAhead={ 4 }
           onRenderCell={ (item, index) => (
             <div
               className='ms-ListGridExample-tile'
               data-is-focusable={ true }
               style={ {
-                width: this._columnWidth,
-                height: this._rowHeight,
-                backgroundColor: `rgba(${ 4 * (index % 32) + 127 }, ${ 4 * (index % 32) + 127 }, ${ 4 * (index % 32) + 127 }, 1)`
+                width: (100 / this._columnCount) + '%'
               } }>
-              <Image
-                className='ms-ListGridExample-image'
-                src={ this._getThumbnail(item, this._rowHeight) }
-                imageFit={ ImageFit.cover }
-                width={ this._rowHeight }
-                height={ this._rowHeight } />
-              <span className='ms-ListGridExample-label'>
-              { `item ${ index }` }
-              </span>
+                <div className='ms-ListGridExample-sizer'>
+                  <div className='msListGridExample-padder'>
+                    <img src={ item.thumbnail } className='ms-ListGridExample-image' />
+                    <span className='ms-ListGridExample-label'>
+                    { `item ${ index }` }
+                    </span>
+                  </div>
+                </div>
             </div>
           ) }
         />
       </FocusZone>
     );
   }
 
-  private _getThumbnail(item, rowHeight) {
-    const aspectRatio = item.width / item.height;
-
-    if (item.width >= item.height) {
-      return `//placekitten.com/${ Math.round( aspectRatio * rowHeight) }/${ rowHeight }`;
-    } else {
-      return `//placekitten.com/${ rowHeight }/${ Math.round(rowHeight / aspectRatio) }`;
-    }
-  }
-
   private _getItemCountForPage(itemIndex: number, surfaceRect) {
     if (itemIndex === 0) {
-      this._columnCount = Math.ceil(surfaceRect.width / 200);
+      this._columnCount = Math.ceil(surfaceRect.width / MAX_ROW_HEIGHT);
```

Comment:
No, because the column count is derived by the ceiling of the total surface width divided by the max row height, not divided by the max column width.